### DISCUSSION TITLE
看文档，是按照页面进行抓取的，没有基于域名的抓取？

### DISCUSSION THREAD

**Discussion by laoxiong** [coder-hxl/x-crawl#134] [General]
看文档 https://coder-hxl.github.io/x-crawl/cn/guide/crawl-page ， 

是按照页面进行抓取的，没有基于域名的抓取？如深度优先 or 广度优先一类的。

**Comment by coder-hxl** [comment]
这个没有，有一部分原因是因为：https://coder-hxl.github.io/x-crawl/cn/about/faq#using-crawlpage-api-causes-the-program-to-crash
